### PR TITLE
Some styling and adding spotify links and preview

### PR DESCRIPTION
**Pull request by tonymelek**

### CODE REVIEW

**Review by smoneo — CHANGES_REQUESTED** [review]
please pull to update current bandsintown file

**Review by chrisjmckeown — APPROVED** [review]
LGTM

**Review by chrisjmckeown — APPROVED** [review]
LGTM

File: Assets/JS/bandsintown.js [hunk 1]
```diff
@@ -35,46 +38,78 @@ $(document).ready(function() {
...
 
-    function appendArtistInfo(data) {
+    function appendArtistInfo(data, artist) {
         // artist info
         if (data.name) {
             $(".bands-in-town-list").append($("<h2>").text(data.name).attr("style", "margin: 0 0 5px 0"));
-        }
+        } else {
+            $(".bands-in-town-list").append($("<p>").text(artist + " is not in town"))
+        };
```
Comment (smoneo) [inline Assets/JS/bandsintown.js:113]: data.name will always be called if the artist exists in bandsintown and doesn't have any events. The else statement should be after data.upcoming_event_count as the error being undefined only occurs when the artist does not have any events
```suggestion
        }
```

File: Assets/JS/bandsintown.js [hunk 2]
```diff
@@ -35,46 +38,78 @@ $(document).ready(function() {
...
         if (data.upcoming_event_count) {
             $(".bands-in-town-list").append($("<p>").text(data.upcoming_event_count + " upcoming events"));
-        } else {$(".bands-in-town-list").append($("<p>").text(data.name + " is not in town"))};
+        }
```
Comment (smoneo) [inline Assets/JS/bandsintown.js:116]:
```suggestion
        } else {$(".bands-in-town-list").append($("<p>").text(data.name + " is not in town"))};
```

File: Assets/JS/bandsintown.js [hunk 3]
```diff
@@ -35,46 +38,78 @@ $(document).ready(function() {
...
         artists_search_results.forEach((element, index) => {
-            if (parseInt(this.id) == index) {
+            if (parseInt(this.id) === index) {
                 displayBandsInTownData(element)
             }
         });
     })
 
-    function appendArtistInfo(data) {
+    function appendArtistInfo(data, artist) {
```
Comment (smoneo) [inline Assets/JS/bandsintown.js:107]:
```suggestion
    function appendArtistInfo(data) {
```

**Review by chrisjmckeown — APPROVED** [review]
LGTM

**Review by chrisjmckeown — APPROVED** [review]
LGTM

**Review by smoneo — CHANGES_REQUESTED** [review]

File: Assets/JS/bandsintown.js (hunk 1 above)
Comment (chrisjmckeown) [reply Assets/JS/bandsintown.js:113]: LGTM

**Review by smoneo — APPROVED** [review]
LGTM

**Pull request merged**